### PR TITLE
IDE-4396 add watch workspace tests and sf

### PR DESCRIPTION
**Pull request by rui123wang**
Testing,don't merge,thanks

### CODE REVIEW

**Comment by yanan** [comment]
Functional Tests build Failed.
Test Failed on http://ide-build-1:9090/job/Test-Checker//194

**Comment by rui123wang** [comment]
ft retest

**Comment by yanan** [comment]
Functional Tests build Failed.
Test Failed on http://ide-build-1:9090/job/Test-Checker//195

File: functional-tests/common/com.liferay.ide.functional.liferay/src/com/liferay/ide/functional/liferay/action/ViewAction.java [hunk 1]
```diff
@@ -596,10 +608,16 @@ public void stop(String serverLabel) {
...
 
-		public void stopWatchingProject(String serverName, String workspaceName, String... modules) {
-			_getServers().selectTreeItem(serverName, workspaceName, modules[0]);
+		public void stopWatchingProject(String serverName, String workspaceName) {
+			_getServers().selectTreeItem(serverName, workspaceName);
+
+			_getServers().contextMenu(true, STOP_WATCHING_PROJECT, serverName, workspaceName);
+		}
+
+		public void stopWatchingProject(String serverName, String workspaceName, String module) {
```
Comment (yanan) [inline functional-tests/common/com.liferay.ide.functional.liferay/src/com/liferay/ide/functional/liferay/action/ViewAction.java]: rename module to project

File: functional-tests/common/com.liferay.ide.functional.liferay/src/com/liferay/ide/functional/liferay/action/ViewAction.java [hunk 2]
```diff
@@ -645,6 +663,21 @@ public boolean visibleServer(String serverName) {
 			return _getServers().isVisibleStartsBy(serverName);
 		}
 
+		public boolean visibleWorkspaceTry(String serverLabel, String workspaceName, String... modules) {
```
Comment (yanan) [inline functional-tests/common/com.liferay.ide.functional.liferay/src/com/liferay/ide/functional/liferay/action/ViewAction.java]: modules is not actually used

File: functional-tests/tools/module/module.watch/module.watch.tomcat.71/src/com/liferay/ide/functional/module/tests/WatchModuleLiferayWorkspaceGradleTomcatTests.java [hunk 3]
```diff
@@ -63,28 +68,87 @@ public void watchMVCPortlet() {
 	}
 
 	@Test
-	public void watchWarMvcPortlet() {
+	public void watchMvcWarPortlet() {
```
Comment (yanan) [inline functional-tests/tools/module/module.watch/module.watch.tomcat.71/src/com/liferay/ide/functional/module/tests/WatchModuleLiferayWorkspaceGradleTomcatTests.java]: why we change this to MvcWar?

**Comment by yanan** [comment]
Please see the comments and resend, thanks

**Pull request closed without merging**